templates/docs/examples/patterns/navigation/sliding.html

```diff
@@ -0,0 +1,200 @@
+                </button>
+              </li>
+              <li>
+                <a href="#" class="p-navigation__dropdown-item">Introduction</a>
+              </li>
+              <li>
+                <a href="#" class="p-navigation__dropdown-item">News</a>
+              </li>
+              <li>
+                <a href="#" class="p-navigation__dropdown-item">Getting started - Command line</a>
+              </li>
+              <li>
+                <a href="#" class="p-navigation__dropdown-item">Getting started - OpenStack</a>
+              </li>
+              <li>
+                <a href="#" class="p-navigation__dropdown-item">Getting started - OpenNebula</a>
+              </li>
+            </ul>
+          </li>
+          <li class="p-navigation__item--dropdown-toggle" id="link-2">
+            <button href="#link-2-menu" aria-controls="link-2-menu" class="p-navigation__link">
+              LXCFS
+            </button>
+            <ul class="p-navigation__dropdown" id="link-2-menu" aria-hidden="true" data-level="1">
+              <li class="p-navigation__item--dropdown-close" id="link-2-back">
+                <button href="#link-2-menu" aria-controls="link-2-menu" class="p-navigation__link js-back">
+                  Back
+                </button>
+              </li>
+              <li>
+                <a href="#" class="p-navigation__dropdown-item">Introduction</a>
+              </li>
+              <li>
+                <a href="#" class="p-navigation__dropdown-item">News</a>
+              </li>
+              <li>
+                <a href="#" class="p-navigation__dropdown-item">Getting started</a>
+              </li>
+              <li class="p-navigation__item--dropdown-toggle" id="link-4">
+                <button href="#link-4-menu" aria-controls="link-4-menu" class="p-navigation__link">
+                  Nested Layer
+                </button>
+                <ul class="p-navigation__dropdown" id="link-4-menu" aria-hidden="true" data-level="2">
+                  <li class="p-navigation__item--dropdown-close" id="link-4-back">
+                    <button href="#link-4-menu" aria-controls="link-4-menu" class="p-navigation__link js-back">
+                      Back
+                    </button>
+                  </li>
+                  <li>
+                    <a href="#" class="p-navigation__dropdown-item">Introduction</a>
+                  </li>
+                  <li>
+                    <a href="#" class="p-navigation__dropdown-item">News</a>
+                  </li>
+                  <li>
+                    <a href="#" class="p-navigation__dropdown-item">Getting started</a>
+                  </li>
+                </ul>
+              </li>
+            </ul>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </div>
+</header>
+
+<style>
+  body { margin: 0; }
+</style>
+
+<script>
+  document.querySelector('.js-menu-button').addEventListener('click', function(e) {
```

Comment:
Ideally this should be separated to a script that is included here.

Does this only cover mobile animation, or also desktop dropdowns? It would be good if there is only one script that covers all navigation needs.

Reply:
It does everything! I'll abstract it to its file